DeepCat.js

```diff
@@ -562,14 +565,32 @@
 	}
 
 	function addSmallFormHint() {
-		var smallHintBox = '<div id="deepcat-smallhint">'
+		var smallHintBox = '<div id="deepcat-smallhint" style="display: none;">'
 			+ '<img id="deepcat-smallhint-hide" title="' + mw.msg( 'deepcat-smallhint-close' ) + '" src="https://upload.wikimedia.org/wikipedia/commons/4/44/Curation_bar_icon_close.png">'
 			+ mw.msg( 'deepcat-hintbox-small' )
 			+ '</div>';
 		$( '#searchform' ).after( smallHintBox );
 		$( '#deepcat-smallhint-hide' ).on( 'click', hideSmallHint );
 	}
 
+	function initializeHintCodeIfNeeded() {
```

Comment:
On first glimpse at the code I am really confused what "hint code" is.

Looking at the method what it does, it loads css styles and adds some HTML elements. That's OK but which of those two is the actual "code"? Or is the "hint code" something different hidden somewhere deeper?
Wouldn't `initializeHintBoxes(IfNeeded)?` sound more clear in this case?